extensions/mssql/src/mssql.d.ts

```diff
@@ -554,6 +554,41 @@ declare module 'mssql' {
 		 * @param projectUri Absolute path of the project, including .sqlproj
 		 */
 		getSqlObjectScripts(projectUri: string): Promise<GetScriptsResult>;
+
+		/**
+		 * Add a SQL object script to a project
```

Comment:
None items aren't necessarily SQL object scripts, like a publish profile is added as a None item

Reply:
True.  I think these comments are from Tools Service, so I'll correct them over there.

"NoneItem" is probably a more accurate name; I'll shift it that way